emtable/metadata.py

```diff
@@ -227,10 +227,10 @@ def _findLabelLine(self, inputFile):
 
         rawLine = inputFile.readline()
         while rawLine:
-            if rawLine.startswith('_'):
+            if rawLine.lstrip().startswith('_'):
```

Comment:
Is there any reason why we don't just do rawLine = inputFile.readline().strip() ?

Reply:
good point! I guess I was distracted by "rawLine" name